### DISCUSSION TITLE
23.03.16 Diary Weather component 진행상황 (이원준)

### DISCUSSION THREAD

**Discussion by copiCat0** [withPet-team/withPet-client#67] [Report]
# 🗒️ Description
20230316 Diary 날씨 컴포넌트 진행상황입니다.

# ✅ Todo 및 진행 상황
앞으로 해야할 일
- [x] 스프라이트 이미지
- [x] 클릭시 이미지 변경 기능
- [x] 상태관리 리덕스 툴킷으로 작성
- [ ] 컴포넌트 랜더링 후 테스트

# ETC
랜더링이 안되서 테스트를 못하고 있습니다. 도와주실 수 있나요..?<br>
![image](https://user-images.githubusercontent.com/80018207/226089089-9269a2f6-1d14-4883-9bad-b5d530ddc6f5.png)
<br>
에러뜨는건 따로 없는데, 승연님꺼 pull 받고나서도 빈 화면이 나오는데, 어디가 잘못됬는지 찾기가 어렵습니다 ㅠㅠㅠ

url : github.com/copiCat0/withPet-client<br>
branch : feature/diary

**Comment by jaesukpark77** [comment]
혹시 경로를 /diary로 적용해보셨나요???

**Reply by copiCat0** [reply]
npm start /diary 이렇게 말씀이신가요?

**Reply by jaesukpark77** [reply]
localhost:3000/diary로 적용해보셨나요???

**Reply by copiCat0** [reply]
아하 됬어요.. 안해본 티가 팍팍나네요ㅋㅋ하 ㅜㅜ

**Reply by jaesukpark77** [reply]
아닙니다!!! 저도 배우고 있습니다 ㅎㅎㅎ

**Reply by copiCat0** [reply]
감사합니다 ㅎㅎ 👍
